src/translator/sanelogging.h

```diff
@@ -0,0 +1,44 @@
+#ifndef SRC_BERGAMOT_SANELOGGING_H_
+#define SRC_BERGAMOT_SANELOGGING_H_
+
+#include "spdlog/spdlog.h"
+#include <iostream>
+
+namespace marian {
+
+#define PLOG(worker, level, ...)
```

Comment:
What is PLOG?  You said per-worker logging.  Can I have a comment to that effect?

Reply:
Or you could remove if not needed anymore.  Or comment that it is for debugging only.